lang/cs/Org.Apache.REEF.Driver/Bridge/DriverBridge.cs

```diff
@@ -323,6 +312,17 @@ public ulong[] Subscribe()
             return handlers;
         }
 
+        [Obsolete]
```

Comment:
Please add some documentation here. The goal is to make the life of the person removing this code in 0.13 easier. At the minimum, reference the JIRA and the version in which this was obsoleted.